src/test/java/api/support/fixtures/ScheduledNoticeProcessingTimerClient.java

```diff
@@ -0,0 +1,25 @@
+package api.support.fixtures;
+
+import static api.support.RestAssuredClient.post;
+
+import java.net.URL;
+
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeUtils;
+
+import api.support.APITestContext;
+import io.vertx.core.json.JsonObject;
+
+public class ScheduledNoticeProcessingTimerClient {
```

Comment:
Why does this have `timer` in the name? As far as I can tell, this does not utilise any kind of timing?

Reply:
Agree. I was thinking about timer interface, but now it seems to me to be redundant. I'll change it to  `ScheduledNoticeProcessingClient `

Reply:
Done